lizmap/www/assets/js/map.js

```diff
@@ -6742,7 +6734,7 @@ lizMap.events.on({
        // IGN base layers
         if ('ignKey' in evt.config.options){
           var ignKey = evt.config.options.ignKey;
```

Comment:
vscode me dit ici : `'ignKey' is declared but its value is never read.ts(6133)`

Je vais ajouter un projet QGIS pour tester manuellement les fonds de carte. STP trouve un moyen d'avoir une clé pour tester que le scan25 fonctionne.